crates/gpui/src/text_system.rs

```diff
@@ -43,6 +43,9 @@ pub struct FontFamilyId(pub usize);
 
 pub(crate) const SUBPIXEL_VARIANTS: u8 = 4;
 
+#[cfg(target_os = "windows")]
+pub(crate) const SUBPIXEL_VARIANTS_Y: u8 = 1;
```

Comment:
Can we have something like
```suggestion
pub(crate) const SUBPIXEL_VARIANTS_X: u8 = 4;

pub(crate) const SUBPIXEL_VARIANTS_Y: u8 = if cfg!(target_os = "windows") {
    1
} else {
    SUBPIXEL_VARIANTS_X
};
```
and get rid of the platform-specific logic in `window.rs`? I will be porting these text rendering changes to Linux so it'll need to change either way.

Reply:
yeah that seems reasonable